tasks/system/general/setup_xfs.yml

```diff
@@ -36,6 +36,12 @@
     fstype: xfs
     dev: /dev/lxc/data
 
+- name: remove automount of /dev/{{ device_name }}
```

Comment:
It's strange that we can create the volume a few lines before with the device mounted.
Could it be that something is adding this mount lather on ? 
I feel that we should dig a bit more to know how it gets there, or at least be sure that it doesn't get re-added.

Reply:
And that problem occurs only with ubuntu, not SLES (and all the system tasks in this playbook are common, only docker installation differs a bit: apt instead of zypper)